runtime/syntax/ignore.yaml

```diff
@@ -0,0 +1,14 @@
+filetype: 'ignore'
+
+detect:
+  filename: '.*ignore$'
+
+rules:
+  - statement: '.*$'         # files
+  - constant.string: '.*/$'  # diretories
+  - error: "^\\!.*"          # negated
+
+  - special: '/'
+  - identifier: '!|\*|\[.*]'
```

Comment:
this should probably also include `?` (any single character)

Reply:
Based on your suggestions I thought of something like this, what do you think?
```yaml
---
filetype: ignore

detect:
  filename: .*ignore$

rules:
  - statement: .*$           # files
  - constant.string: .*/$    # diretories
  - diff-deleted: '^\!.*'    # negated

  - special: /
  - identifier: '!|\?|\*|\[.*?]|\\.'

  - comment: '#.*$'
```

Reply:
Are the colors strange?